### ISSUE TITLE
[MNG-4986] After "mvn deploy" an SNAPSHOT artifact to company central repo(artifactory-2.3.1), Could not download the artifact in other machines.

### ISSUE THREAD

**Issue by jira-importer** [apache/maven#6910]
**[Isaac Qu](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=quyanwei@sina.com)** opened **[MNG-4986](https://issues.apache.org/jira/browse/MNG-4986?redirect=false)** and commented

After "mvn deploy" an SNAPSHOT artifact to company central repo(artifactory-2.3.1), Could not download the artifact in other machines.

MAVEN 2.2.1 has no this problem.

Following is detailed error info.

D:\work_p2p\tcpip>mvn install
[INFO] Scanning for projects...
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Building Wacube Mebula TCP/IP Service 1.1.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
Downloading: http://192.168.0.2:8080/artifactory/repo/com/wacube/mebula/com.wacube.mebula.buffer/maven-metadata.xml
Downloading: http://192.168.0.2:8080/artifactory/repo/com/wacube/mebula/com.wacube.mebula.buffer/maven-metadata.xml
Downloaded: http://192.168.0.2:8080/artifactory/repo/com/wacube/mebula/com.wacube.mebula.buffer/maven-metadata.xml (337
B at 2.3 KB/sec)
Downloaded: http://192.168.0.2:8080/artifactory/repo/com/wacube/mebula/com.wacube.mebula.buffer/maven-metadata.xml (337
B at 1.9 KB/sec)
[WARNING] The POM for com.wacube.mebula:com.wacube.mebula.buffer:jar:1.0.1-SNAPSHOT is missing, no dependency informatio
n available
Downloading: http://192.168.0.2:8080/artifactory/repo/com/wacube/mebula/com.wacube.mebula.core/maven-metadata.xml
Downloading: http://192.168.0.2:8080/artifactory/repo/com/wacube/mebula/com.wacube.mebula.core/maven-metadata.xml
Downloaded: http://192.168.0.2:8080/artifactory/repo/com/wacube/mebula/com.wacube.mebula.core/maven-metadata.xml (375 B
at 3.4 KB/sec)
Downloaded: http://192.168.0.2:8080/artifactory/repo/com/wacube/mebula/com.wacube.mebula.core/maven-metadata.xml (375 B
at 2.9 KB/sec)
[WARNING] The POM for com.wacube.mebula:com.wacube.mebula.core:jar:1.1.1-SNAPSHOT is missing, no dependency information
available
[WARNING] The POM for com.wacube.mebula:com.wacube.mebula.core:jar:1.1.2-SNAPSHOT is missing, no dependency information
available

---

**Affects:** 3.0, 3.0.1, 3.0.2

**Attachments:**
- [buffer-1.0.1-SNAPSHOT.zip](https://issues.apache.org/jira/secure/attachment/12713149/buffer-1.0.1-SNAPSHOT.zip) (_64.89 kB_)
- [local-repo-contents.zip](https://issues.apache.org/jira/secure/attachment/12713251/local-repo-contents.zip) (_1.88 kB_)
- [maven-debug-inf.txt](https://issues.apache.org/jira/secure/attachment/12713292/maven-debug-inf.txt) (_38.43 kB_)

1 votes, 2 watchers

**Comment by jira-importer** [comment]
**[Isaac Qu](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=quyanwei@sina.com)** commented

Thanks a lot to Benjamin Bentmann. I read you comment and tried, but the issue still exists.
I upload the debug(mvn -X) and the local repo.

**Comment by jira-importer** [comment]
**[Benjamin Bentmann](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=bentmann)** commented

The log and the repo contents you provided don't match up. The log says

```
[DEBUG] Could not find metadata com.wacube.mebula:com.wacube.mebula.buffer:1.0.1-SNAPSHOT/maven-metadata.xml in local (C:\devtools\apache-maven-repo\repository)
```

while the `buffer-1.0.1-SNAPSHOT.zip` contains `maven-metadata-local.xml`. Likewise

```
[WARNING] The POM for com.wacube.mebula:com.wacube.mebula.buffer:jar:1.0.1-SNAPSHOT is missing
```

doesn't match the fact that the `_maven.repositories` marks the file as locally present. Without an example project, I'm not able to draw a fruitful conclusion from this.

**Comment by jira-importer** [comment]
**[Isaac Qu](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=quyanwei@sina.com)** commented

This issue fixed in 3.0.3, maybe the cause of this issue is equal to MNG-4987.
